installers/ansible/roles/pgo-operator/files/crds/perconapgclusters-crd.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: perconapgclusters.crunchydata.com
```

Comment:
maybe it is better to use perconapgclusters.pg.percona.com

Reply:
I think we should do this in separate PR after merge

Reply:
Yeah, I agree we should rename it.